client/src/public/ts/popup-notification.ts

```
@@ -0,0 +1,66 @@
+    this.id = PopupNotification.nextID;
+    this.title = title;
+    this.text = text;
+    this.buttonStyle = buttonStyle || PopupNotificationButtonStyle.SINGLE;
+    this.render();
+  }
+
+  // TODO: wtf is this
+  render() {
+    PopupNotification.activeNotifications++;
+    PopupNotification.activeNotificationIDs.push(this.id);
+    let buttons = `<button id="popup-notification--${this.id}__close-button">Close</button>`;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'buttons' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)